### PR TITLE
Clarify Ideal Staking rate and add ref links

### PR DESCRIPTION
**Pull request by DrW3RK**
Also, fetch the current issuance from the chain state

### CODE REVIEW

File: docs/learn/learn-staking.md [hunk 1]
```diff
@@ -601,20 +599,22 @@ dilute over time.
 :::
 
 The ideal staking rate on Polkadot also varies with the number of parachains (50% is the current
-estimation of all DOT that should be staked, per parachain slot).
+estimation of all DOT that should be staked when all the parachain slots are taken).
```
Comment (profradha) [inline docs/learn/learn-staking.md]: Needs more clarity. Is it 50% when running 1 parachain? or is it 50% when running parachains at capacity?

Comment (filippoweb3) [reply docs/learn/learn-staking.md]: According to the w3f research article is 50% when running parachains at capacity, 75% with no parachains.

File: docs/learn/learn-staking.md [hunk 2]
```diff
@@ -623,7 +623,7 @@ in the percentage of staking rewards that go to the Treasury.
 - **Blue line**: Inflation rewards to stakers
```
Comment (filippoweb3) [inline docs/learn/learn-staking.md]:
```suggestion
- **Blue line**: Annual inflation rate of NPoS, i.e. total amount of tokens minted to pay validators and nominators.
```

File: docs/learn/learn-staking.md [hunk 3]
```diff
@@ -623,7 +623,7 @@ in the percentage of staking rewards that go to the Treasury.
...
 - **Green line**: Staker rate of return
```
Comment (filippoweb3) [inline docs/learn/learn-staking.md]:
```suggestion
- **Green line**: Annual rewards rate for stakers. For instance, 0.2 corresponds to 20% of annual returns on the staked tokens
```

**Review by filippoweb3 — COMMENTED** [review]
Changed the definitions of the two lines in the figure. I think it is still a bit hard to follow in some points without reading the w3f research article, which might be too technical for some readers. Maybe I will propose some additional text in the staking PR?

File: docs/learn/learn-staking.md (hunk 3 above)
Comment (profradha) [reply docs/learn/learn-staking.md]: What do you think of adding a numeric example for better understanding for a generic reader?

File: docs/learn/learn-staking.md (hunk 1 above)
Comment (profradha) [reply docs/learn/learn-staking.md]: In the current implementation, the ideal staking rate is dynamic and adjusts itself as the number of slots on the relaychain increase. Will summarize that in the doc

https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L535

File: docs/learn/learn-staking.md [hunk 4]
```diff
@@ -601,29 +599,31 @@ dilute over time.
...
+![staking](../assets/NPoS/staking-rate-with-parachains.png)
 
 <p style={{textAlign:"center"}}>Source: <a href="https://w3f-research.readthedocs.io/en/latest/polkadot/overview/2-token-economics.html">Research - Web3 Foundation</a></p>
 
 - **x-axis**: Proportion of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} staked
 - **y-axis**: Inflation, annualized percentage
-- **Blue line**: Inflation rewards to stakers
-- **Green line**: Staker rate of return
+- **Blue line**: Annual inflation rate of NPoS, i.e. total amount of tokens minted to pay validators and nominators.
+- **Green line**: Annual rewards rate for stakers. For instance, 0.2 corresponds to 20% of annual returns on the staked tokens
```
Comment (filippoweb3) [inline docs/learn/learn-staking.md]:
```suggestion
- **Green line**: Annual rewards rate for stakers. For instance, 0.2 corresponds to 20% of annual returns on the staked tokens.
```

**Review by filippoweb3 — APPROVED** [review]
I think everything is clear. I will probably do a check on the staking page when the current staking PRs are merged. Thank you for asking my review Radha!
